### ISSUE TITLE
Does not build out-of-the-box

### ISSUE THREAD

**Issue by joergplewe** [ryansolid/solid#118]
Hi!

Following the documentation for v0.16.6 on Windows10 I expected this to work:
```
npm init solid app my-app
cd .\my-app\
npm run build
```

Instead, I get an error:
```
./src/index.js
Attempted import error: './components/App' does not contain a default export (imported as 'App').
```

`npm run start` in fact works as expected. What am I doing wrong?

The full log is as follows:

```
PS C:\dev> npm init solid app my-app
npx: installed 91 in 10.636s

Creating a new Solid app in C:\dev\my-app.

Installing packages. This might take a couple of minutes.
Installing solid-js, and solid-scripts...


> core-js@2.6.11 postinstall C:\dev\my-app\node_modules\babel-runtime\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@3.6.4 postinstall C:\dev\my-app\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

+ solid-js@0.16.6
+ solid-scripts@0.0.28
added 1303 packages from 710 contributors and audited 907722 packages in 90.365s
found 0 vulnerabilities


Initialized a git repository.

Success! Created my-app at C:\dev\my-app
Inside that directory, you can run several commands:

  npm start
    Starts the development server.

  npm run build
    Bundles the app into static files for production.
    Starts the test runner.

We suggest that you begin by typing:

  cd C:\dev\my-app
  npm start

Happy hacking!
PS C:\dev> cd .\my-app\
PS C:\dev\my-app> npm run build

> my-app@0.1.0 build C:\dev\my-app
> solid-scripts build

Creating an optimized production build...
Failed to compile.

./src/index.js
Attempted import error: './components/App' does not contain a default export (imported as 'App').


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! my-app@0.1.0 build: `solid-scripts build`
npm ERR! Exit status 1
```

**Comment by ryansolid** [comment]
Thanks for reporting.. I was recently changing how HMR worked and seemed to have broken the build. Give me a min I will get that fixed right up.

**Comment by ryansolid** [comment]
Ok found it and have pushed the fix. I was not handling the production build bypass properly using  `export * from ....` instead of `export { default } from ...`. I was having some issues with it caching older versions of sub deps so I have updated solid-scripts instead. You will need to update solid-scripts in your package.lock to version 0.0.29 and reinstall the node modules.

.. NPM is taking forever to update, I published the new version like 20 mins ago.. just waiting to confirm.
